test/e2e/service.go

```diff
@@ -34,11 +34,15 @@ import (
 
 var _ = Describe("Services", func() {
 	var c *client.Client
+	// Use these in tests.  They're unique for each test to prevent name collisions.
+	var namespace0, namespace1 string
 
 	BeforeEach(func() {
 		var err error
 		c, err = loadClient()
 		Expect(err).NotTo(HaveOccurred())
+		namespace0 = "e2e-ns-" + dateStamp() + "-0"
```

Comment:
As for deleting the namespaces, yes, I will add some additional cleanup code in AfterEach(), but that requires a PR that derekwaynecarr@ is still working on (I believe), to allow me to delete entire namespaces.